### DISCUSSION TITLE
Incident Response workflow with SO2?

### DISCUSSION THREAD

**Discussion by n0th1nt0lOOz** [Security-Onion-Solutions/securityonion#2060] [Unsupported Versions] [answered]
Hi Mates,

After finish installing the Master and Sensor node. I very confusing about the new workflow and don't know what I will do.
With version 16, I open squert, check the summary and views, then check the event tab and choose the suitable response to each event.
With SO2, after sending the alert to TheHive, I don't know what to do next. Empty example cases or responders. Don't know how to add/run a command/script from TheHive to block IP/Host,...

Could anyone provide a general guide about the new workflow and how to respond to incidents? I read the document but not really understand how components link and work with others.

Thanks

**Comment by weslambert** [comment, marked as answer]
We have some videos speaking to the general workflow here: 

https://www.youtube.com/playlist?list=PLljFlTO9rB155aYBjHw2InKkSMLuhWpxH

Let me know if you have any more pointed questions.

**Reply by n0th1nt0lOOz** [reply]
Hi I watched that playlist. But I don't see any step guide for response to an event after investigate. Like the responder in TheHive. What I need to configure if I want to run a script with the argument is the IP of that event? I see the Cortex integrated but no guide to configure for incident response.
Thanks

**Reply by weslambert** [reply]
This is really more of a TheHive question, and really specific to particular Cortex analyzers/responders, so you may want to consult their documentation. I'll look at adding some examples more pertinent to SO soon.

**Reply by n0th1nt0lOOz** [reply]
Yes, please!
I can't make any responder because cortex responder configuration and script directory are not mentioned in the document.

**Reply by n0th1nt0lOOz** [reply]
> This is really more of a TheHive question, and really specific to particular Cortex analyzers/responders, so you may want to consult their documentation. I'll look at adding some examples more pertinent to SO soon.

Hi, I give you some info about pulling analyzer & responders from Cortex community. I follow the Cortex instruction but I see that SO has a different structure. Please review and make SO can pull analyzer & responders from Cortex community instead of creating new manually
https://blog.thehive-project.org/2020/03/25/cortex-analyzers-2-6-0-146-analyzers-18-responders/

Thanks & Regards
